sudachi/src/dic/dictionary.rs

```diff
@@ -86,7 +82,7 @@ impl JapaneseDictionary {
     ) -> SudachiResult<JapaneseDictionary> {
         let mut basic_dict = LoadedDictionary::from_system_dictionary(
             unsafe { storage.system_static_slice() },
-            &cfg.character_definition_file,
+            cfg.complete_path(&cfg.character_definition_file)?.as_path(),
```

Comment:
How about to add `cfg.resolved_character_definition_file()`?

Reply:
I think that special methods should focus on dictionary paths and other auxiliary files should use `complete_path`. Also, I want to incorporate character definitions into the new dictionary format.